lib/python3.13/site-packages/_pytest/assertion/truncate.py

```
@@ -0,0 +1,116 @@
+from _pytest.assertion import util
+from _pytest.config import Config
+from _pytest.nodes import Item
+
+
+DEFAULT_MAX_LINES = 8
+DEFAULT_MAX_CHARS = 8 * 80
+USAGE_MSG = "use '-vv' to show"
+
+
+def truncate_if_required(
+    explanation: List[str], item: Item, max_length: Optional[int] = None
```

Ruff (0.15.2)

[warning] Unused function argument: `max_length`

(ARG001)